content/tutorial-deep-reinforcement-learning-with-pong-from-pixels.ipynb

```diff
@@ -0,0 +1,1275 @@
+{
```

Comment:
<p>I think either something is missing here or the : at the end of the line is misplaced :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
It was supposed to be a full stop. Fixed. Thanks for spotting it!